contracts/Token.sol

```diff
@@ -23,6 +23,8 @@ contract Token {
     // A mapping is a key/value map. Here we store each account balance.
     mapping(address => uint256) balances;
 
+    event Transfer(address to, uint256 amount);
```

Comment:
I think we should use the ERC-20 definition:

```solidity
event Transfer(address indexed _from, address indexed _to, uint256 _value)
```

Otherwise we are setting our users in the wrong direction.

Reply:
Addressed in https://github.com/NomicFoundation/hardhat-boilerplate/pull/61/commits/6acfc9b9f9c4f15d096640bec3b5517f435c8ac3